<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>240</width>
      <height>154</height>
     </rect>
    </property>
    <property name="title">
     <string>LOGIN</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_username">
         <property name="text">
          <string>USERNAME</string>
         </property>
         <property name="buddy">
          <cstring>line_username</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_username"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>PASSWORD</string>
         </property>
         <property name="buddy">
          <cstring>line_password</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_password"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="sign_in">
       <property name="text">
        <string>SIGN IN</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menulogin">
    <property name="title">
     <string>login</string>
    </property>
   </widget>
   <addaction name="menulogin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
